lib/terratag-action.spec.ts

```diff
@@ -44,7 +48,7 @@ describe('terratag action', () => {
     });
     it('should download terratag from expected url', () => {
       expect(mockedTC.downloadTool.mock.calls).toEqual([
-        ['https://github.com/env0/terratag/releases/download/v1.2.3/terratag_1.2.3_linux_amd64.tar.gz']
+        [`https://github.com/env0/terratag/releases/download/v1.2.3/terratag_1.2.3_${osPlatform}_amd64.tar.gz`]
```

Comment:
The cpu architecture may also be different:
```suggestion
        [`https://github.com/env0/terratag/releases/download/v1.2.3/terratag_1.2.3_${osPlatform}_${os.os.arch()}.tar.gz`]
```

Reply:
ab7fa460886ed211c5db8cc2ae4bbcf4bceecfde
had to use `mapOS` and `mapArch` for some reason all our terratag tgz files have `amd64` as arch